apps/src/lab2/views/components/Instructions/ResourcePanel/validationTourHelpers.ts

```diff
@@ -0,0 +1,21 @@
+import lab2I18n from '@cdo/apps/lab2/locale';
+
+export const VALIDATION_TOUR_STEPS = [
+  {
+    element: '#resource-panel-tab-validation',
```

Comment:
Should these constants be shared so the React components can reference them directly?